feature-libs/product-configurator/rulebased/occ/variant/converters/occ-configurator-variant-normalizer.ts

```diff
@@ -257,6 +262,19 @@ export class OccConfiguratorVariantNormalizer
     images.push(image);
   }
 
+  getPlaceholder(
```

Comment:
By the way by running `yarn build:libs` the following exception is thrown:
![image](https://user-images.githubusercontent.com/61147963/155667383-a639591c-0f97-44a6-8185-48c3043410a9.png)

Reply:
According to `OccConfigurator` model `domainValues` and `intervalInDomain` could be `undefined`

Reply:
That's because of the strict mode issue I mentioned above

Reply:
fixed